Makefile

```diff
@@ -90,6 +90,15 @@ fmt-verify:
 	    exit 1; \
 	fi
 
+.PHONY: gomod-verify
+gomod-verify:
+	$(GO_CMD) mod tidy
+	@out=`git --no-pager diff go.mod go.sum`; \
```

Comment:
This should be enough :)

`git --no-pager diff --exit-code go.mod go.sum`


Can you do the same for verify below?

Reply:
Please check if the latest changes are what you want.